src/line_launcher/mod.rs

```diff
@@ -65,7 +77,8 @@ impl LineLauncher {
                     if beat_message.minus_sixteenths(last_played_note.duration)
                         == last_played_note.start
                     {
-                        midi_message_sender.fire_note_off(last_played_note.note);
+                        midi_message_sender
+                            .fire_note_off(last_played_note.note.step(pitch_offset).unwrap());
```

Comment:
Wmidi `Note`'s have a `.step()` method for adjusting by a certain # of half-steps